src/tests/ftest/container/fill_destroy_loop.py

```diff
@@ -22,18 +25,65 @@ class BoundaryPoolContainerSpace(TestWithServers):
 
     DER_NOSPACE = "-1007"
 
+    def __init__(self, *args, **kwargs):
+        """Initialize a BoundaryPoolContainerSpace object."""
+        super().__init__(*args, **kwargs)
+
+        self.test_loop = 0
+        self.reclaim_props = []
+        self.delta_bytes = 0
+        self.rand_seed = None
+
+    def setUp(self):
+        """Set up each test case."""
+        super().setUp()
+
+        self.test_loop = self.params.get("test_loop", "/run/pool/*", 0)
+        self.reclaim_props = self.params.get("reclaim_props", "/run/pool/*", [])
+
+        delta = self.params.get("delta", "/run/pool/*", "0")
+        self.delta_bytes = human_to_bytes(delta)
+        self.log.info("==> Set pool delta to %s (%i bytes)", delta, self.delta_bytes)
+
+        self.rand_seed = self.params.get("rand_seed", "/run/*", None)
+        if self.rand_seed is None:
+            self.rand_seed = int.from_bytes(os.urandom(8), byteorder='little')
+        self.log.info("==> Set random seed to %i", self.rand_seed)
+        random.seed(a=int.to_bytes(self.rand_seed, 8, byteorder='little'), version=2)
```

Comment:
We've actually started some of the groundwork for this here
https://github.com/daos-stack/daos/blob/7229422791e82c1a2c3bf2ed8d149af13241e643/src/tests/ftest/util/apricot/apricot/test.py#L145-L148
Where you can use `self.random`. It doesn't yet allow seeding from a config, but we could add a `self.params.get` to that existing snippet.
Would you mind instead using the existing `self.random` and updating `test.py` to optionally use a seed from `self.params`? This would be super useful for other tests as well.


Also, I'm not very familiar, but is it really necessary to do something this complex?
`int.from_bytes(os.urandom(8), byteorder='little')`

Reply:
- [ ] Use self.random
- [ ] Update test.py to fix the seed

I am also not an expert with pseudo random generator.
From what I know and remember, using `urandom()` as seed generator instead of the date (default behavior of python) is recommended: reproducibility, distribution of the seed, etc.
However, for our use case, using `randrange()` for generating the seed should perfectly do the job.

Reply:
At the end using `randrange()` is a security flaw https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-13013/34/testReport/ and `random.Random()` is deprecated.
More details on this security flaw could be find at https://docs.openstack.org/bandit/1.2.0/blacklists/blacklist_calls.html#b311-random
Thus, I am going back to my `urandom()` solution which is also the one used by `random.SystemRandom()` for fixing a default seed value.

Reply:
We can ignore that bandit check since this is test code, not production. We have similar ignores already. I.e. the `# nosec` here
```
seed = random.randrange(sys.maxsize)  # nosec
```
But if `urandom()` works and is secure, no issue there